Frends.Confluence.Request/Frends.Confluence.Request/Definitions/Input.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using static Frends.Confluence.Request.Definitions.Constants;
+
+namespace Frends.Confluence.Request.Definitions;
+
+/// <summary>
+/// Input parameters.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Username used to connect to confluence with Basic Auth.
+    /// </summary>
+    /// <example>john.doe@email.com</example>
+    public string Username { get; init; }
+
+    /// <summary>
+    /// ApiToken used to connect to confluence with Basic Auth.
+    /// </summary>
+    /// <example>AKxDuOPvOWAb2g1-HtJmeqnlcztevlXMqDGtsNURL88=2C9A9D12</example>
+    public string ApiToken { get; init; }
+
+    /// <summary>
+    /// Define method of request.
+    /// </summary>
+    /// <example>HttpMethod.Get</example>
+    public HttpMethod HttpMethod { get; init; }
+
+    /// <summary>
+    /// Api version that will be used in a request.
+    /// </summary>
+    /// <example>ApiVersion.V1</example>
+    public ApiVersion ApiVersion { get; init; }
+
+    /// <summary>
+    /// Confluence name
+    /// </summary>
+    /// <example>frends-test</example>
```

Comment:
Lets not use the domain used for testing for example values in docs. Maybe some random org name or just myconfluence